pkg/pipelines/pipelines/pipelines.go

```diff
@@ -48,6 +55,7 @@ func createBuildImageTask(name string) pipelinev1.PipelineTask {
 		},
 		Params: []pipelinev1.Param{
 			createTaskParam("TLSVERIFY", "$(params.TLSVERIFY)"),
+			createTaskParam("BUILD_EXTRA_ARGS", "--label=io.openshift.build.commit.id='$(params.COMMIT_SHA)' --label=io.openshift.build.commit.ref='$(params.GIT_REF)' --label=io.openshift.build.commit.date='$(params.COMMIT_DATE)' --label=io.openshift.build.commit.author='$(params.COMMIT_AUTHOR)' --label=io.openshift.build.commit.message='$(params.COMMIT_MESSAGE)'"),
```

Comment:
I suspect this could be refactored to avoid repetition here.

Perhaps generating the list of args from the set of names that are defined (see @chetan-rns 's comment below).